pkg/cmd/cli/cmd/status.go

```diff
@@ -72,45 +97,56 @@ func RunStatus(f *clientcmd.Factory, out io.Writer) error {
 	if err != nil {
 		return err
 	}
+	o.namespace = namespace
 
-	describer := &describe.ProjectStatusDescriber{K: kclient, C: client, Server: config.Host}
-	s, err := describer.Describe(namespace, "")
-	if err != nil {
-		return err
-	}
+	o.describer = &describe.ProjectStatusDescriber{K: kclient, C: client, Server: config.Host, Suggest: o.suggest}
+
+	o.out = out
 
-	fmt.Fprintf(out, s)
 	return nil
 }
 
-// RunGraph contains all the necessary functionality for the OpenShift cli graph command
-func RunGraph(f *clientcmd.Factory, out io.Writer) error {
-	client, kclient, err := f.Clients()
-	if err != nil {
-		return err
+// Validate validates the options for the Openshift cli status command.
+func (o StatusOptions) Validate() error {
+	if o.hasArgs {
+		return errors.New("no arguments should be provided")
```

Comment:
I think this error should come from `Complete`.  The `Complete` method can't do its work on bad input, so its reasonable to have it produce an error on some forms of bad input.  Putting it there also keeps the `Options` object clean.